src/bp/core/services/dialog/prompts/confirm.ts

```diff
@@ -0,0 +1,59 @@
+import { IO, Prompt, PromptConfig } from 'botpress/sdk'
+import * as sdk from 'botpress/sdk'
+import { extractEventCommonArgs } from 'common/action'
+import yn from 'yn'
+
+class PromptConfirm implements Prompt {
```

Comment:
do we really want to write prompts in the core ? why not in the bultin module ? I meana we want the users to be ble to write their own prompts in their own module, correct ?

Reply:
As I wrote on the PR description, it's on the core temporarily. I moved them to the builtin module in the other PR (this one was big enough)